controller/search_blackbox_test.go

```diff
@@ -929,3 +929,83 @@ func (s *searchBlackBoxTest) TestIncludedParents() {
 	}
 	assert.Equal(s.T(), successCnt, 4)
 }
+
+func (s *searchBlackBoxTest) TestFilterAssigneeNullAfterWIUpdate() {
+	identitiesTestFixtures := tf.NewTestFixture(s.T(), s.DB, tf.Iterations(1), tf.Areas(1), tf.WorkItems(1))
```

Comment:
`identitiesTestFixtures` should be replaced with `fxt` to be consistent. We use `fxt` almost everywhere else

Reply:
Done. 18fb7090b1c1fa8fda4f4145c1322fc8548503ef